backend/src/personnel/personnel.controller.ts

```diff
@@ -77,7 +77,17 @@ export class PersonnelController {
   ) {
     this.logger.log(`${req.method}: ${req.url} - ${req.username}`);
 
-    return await this.personnelService.updatePersonnel(id, personnel, req.role);
+    delete personnel.availability;  // Ensure we don't use this endpoint to update availability
```

Comment:
I think we should not be including availability with a personnel resource at all. It seems to slow things down and in time the availability (if queried for without time params) will continue to grow...

Reply:
(unrelated to your PR just a general thought)

Reply:
Agreed! This is just for a guarantee since its technically in the DTO. I guess we can remove it from the dto..?